script/generate-component-index

```diff
@@ -23,3 +24,22 @@ do
 done
 
 echo "};" >> $target
+
+# Build sass index
+target=./src/styles/components.scss
+
+echo "// Generated by $0" > $target
+echo "@import './colors.scss';" >> $target
+echo "" >> $target
+
+for f in $( find src/components -name '*.scss' | sort )
+do
+    base=$( basename "$f" )
+    echo "@import '../components/$base';" >> $target
+done
+
+for f in $( find src/pages -name '*.scss' | sort )
+do
+    base=$( basename "$f" )
+    echo "@import '../pages/$base';" >> $target
+done
```

Comment:
That would include `src/styles`, which has (eg) color variables that have to be included up-front.